Presentations/03-data-visualization.Rmd

```diff
@@ -129,22 +143,15 @@ Similarly to previous sessions, you can find some references at the end of this
 # Packages
 library(tidyverse)
 library(plotly)
+library(here)
 
 # Load CSV data
-# Replace with where your data is
-
-FolderPath <- file.path("YOUR/FOLDER/PATH")
-
-dataWorkFolder    <- file.path(projectFolder,"DataWork")
-Data              <- file.path(dataWorkFolder,"DataSets")
-finalData         <- file.path(Data,"Final")
-
-whr_panel <- readRDS(file.path(finalData, "whr_panel.RDS"))
+here::here()
 
 ```
 
 ```{r echo = FALSE, include = FALSE}
-whr_panel <- readRDS(file.path(finalData, "whr_panel.RDS"))
+whr_panel <- readRDS(here::here("DataWork", "DataSets", "Final", "whr_panel.Rds"))
```

Comment:
I removed the `here::` part in #63 to make this consistent with how we use other packages. Was there a particular reason why you included it, @RRMaximiliano?

Reply:
There is no conflict with other package, I am just used to use here as here::